2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Configure stats pid to 212111
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Loading settings from /home/jiaqing/.config/wandb/settings
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Loading settings from /home/jiaqing/Desktop/DeepPurposePlusPlus/wandb/settings
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train/beta.py', 'program_abspath': '/home/jiaqing/Desktop/DeepPurposePlusPlus/train/beta.py', 'program': '/home/jiaqing/Desktop/DeepPurposePlusPlus/train/beta.py'}
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_init.py:_log_setup():529] Logging user logs to /home/jiaqing/Desktop/DeepPurposePlusPlus/wandb/run-20240717_185122-dcokceke/logs/debug.log
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_init.py:_log_setup():530] Logging internal logs to /home/jiaqing/Desktop/DeepPurposePlusPlus/wandb/run-20240717_185122-dcokceke/logs/debug-internal.log
2024-07-17 18:51:22,551 INFO    MainThread:212111 [wandb_init.py:init():569] calling init triggers
2024-07-17 18:51:22,552 INFO    MainThread:212111 [wandb_init.py:init():576] wandb.init called with sweep_config: {}
config: {}
2024-07-17 18:51:22,552 INFO    MainThread:212111 [wandb_init.py:init():619] starting backend
2024-07-17 18:51:22,552 INFO    MainThread:212111 [wandb_init.py:init():623] setting up manager
2024-07-17 18:51:22,553 INFO    MainThread:212111 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-07-17 18:51:22,554 INFO    MainThread:212111 [wandb_init.py:init():631] backend started and connected
2024-07-17 18:51:22,557 INFO    MainThread:212111 [wandb_init.py:init():720] updated telemetry
2024-07-17 18:51:22,588 INFO    MainThread:212111 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-07-17 18:51:27,542 ERROR   Thread-1  :212111 [internal_api.py:execute():349] 500 response executing GraphQL.
2024-07-17 18:51:27,542 ERROR   Thread-1  :212111 [internal_api.py:execute():350] {"error":"An internal error occurred. Please contact support."}
2024-07-17 18:51:31,868 WARNING MainThread:212111 [wandb_init.py:init():1185] interrupted
Traceback (most recent call last):
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    return wi.init()
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 756, in init
    result = run_init_handle.wait(
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "/home/jiaqing/miniconda3/envs/graph/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
